README.md

```diff
@@ -44,11 +44,24 @@ If you need to add features to the API, the best is to [`npm link`](https://docs
 Openfisca
 ---------
 
-```sh
+Openfisca relies on Python 2. You should [install it in a `virtualenv`](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv):
+
+```bash
 cd mes-aides-ui
-npm run install-openfisca # ou pip install --upgrade -r openfisca/requirements.txt si vous utilisez un environnement virtuel
+virtualenv  --python=python2 .venv # To create your virtualenv in ./.venv (a hidden folder)
```

Comment:
I find it a bit risky to give instructions for Python 2 when OpenFisca states it only supports Python 2.7. We're in a grey zone.